Cargo.toml

```diff
@@ -40,8 +40,8 @@ color-print = "0.3.5"
 core-foundation = { version = "0.9.4", features = ["mac_os_10_7_support"] }
 crates-io = { version = "0.40.0", path = "crates/crates-io" }
 criterion = { version = "0.5.1", features = ["html_reports"] }
-curl = "0.4.44"
-curl-sys = "0.4.71"
+curl = "0.4.46"
+curl-sys = "0.4.72+curl-8"
```

Comment:
Still not fixed

Reply:
Renovatebot force-pushed my commit 😨😨😨 <https://github.com/rust-lang/cargo/pull/13507/commits/8488aecc423b09443a9c14a1806d3004d6bcb400>

Reply:
Weird, I've force pushed into their branches before

btw a PR is now up for them to stop including metadata